src/Casdoor.Client/Extensions/QueryMapBuilder.cs

```diff
@@ -1,6 +1,6 @@
 namespace Casdoor.Client;
 
-public class QueryMapBuilder
+internal class QueryMapBuilder
 {
     private readonly Dictionary<string, string?> _map = new();
```

Comment:
`Dictionary` may not be necessary here, the `List` is enough.

Reply:
Used dictionary to prevent key duplicates natively. Changed to List<KeyValuePair>

Reply:
The query strings are allowed duplicates in some scenarios, maybe we need not limit this.